### DISCUSSION TITLE
Creating scripts for installation of other desktops

### DISCUSSION THREAD

**Discussion by creepy-pasta101** [Yisus7u7/termux-desktop-xfce#9] [Ideas]
I guess we can add KDE and MATE desktop components as a part of a separate project
I could compile most of the MATE components natively and @librehat is working on Lxqt in Termux

**Comment by librehat** [comment]
LXQt branch is ready and I'll send the PR into X11-packages when current opening ones are merged and the master submodule updated.

KDE Plasma is a no-go because it depends on OpenGL (hardware or not, it needs GL functionality). Though it's possible _some_ KDE Applications can be ported

**Comment by creepy-pasta101** [comment]
@librehat I guess using gl4es would give a window...
We might need to stabilize it... I will try to provide relevant patches

**Reply by JesusChapman** [reply]
Interesting, when the package is available, I'll install lxqt and make a customization, but I think I'll do it in a separate repository, to save space

**Comment by JesusChapman** [comment]
I've read, and soon lxqt will be available, it should be noted that I have no custom qt5 experience, but and I heard that it's better than gtk and you can add transparency effects, I'll make a separate repository (termux-desktop-lxqt) and then I'll start doing it

**Reply by librehat** [reply]
https://manual.lubuntu.me/stable/5/Panel_Desktop_and_Runner.html has some nice user guide for LXQt desktop environment.
You definitely want to also install `kvantum` theme engine (part of the PR). Its themes can be downloaded at https://store.kde.org/browse/cat/123/order/latest/ 
Have fun

**Reply by JesusChapman** [reply]
Thank you, then let's get to work!! 
Well, to a one that is not yet available lxqt....

**Comment by creepy-pasta101** [comment]
@Yisus7u7 I have a bit of experience compiling mate desktop components... They face a bit of problems during cross compilation.. but works fine on-device....
I guess you could create a separate repository for that too

**Reply by JesusChapman** [reply]
@suhan-paradkar

Of course, apart I would love it to soon be available matte, since I'm good at customizing gtk, it would give it a good look similar to native gnome, perhaps with round-tipped themes

**Reply by creepy-pasta101** [reply]
Great👍👍

**Comment by JesusChapman** [comment]
@suhan-paradkar 

![Screenshot_20210525-150535](https://user-images.githubusercontent.com/64093255/119561525-e7ea3100-bd6a-11eb-9a02-8fce06e6c09b.png)

Not done yet

**Comment by JesusChapman** [comment]
![Screenshot_20210525-154156](https://user-images.githubusercontent.com/64093255/119566717-12d78380-bd71-11eb-80a7-74ae1c915689.png)

**Comment by JesusChapman** [comment]
@suhan-paradkar

The repository is almost ready, could you help me by creating a readme in English?, since my English is very bad jajajaja

**Reply by creepy-pasta101** [reply]
Yeah.. I will create a separate readme taking your readme as a template

**Comment by JesusChapman** [comment]
Pd: I gave you access to the repository

**Reply by creepy-pasta101** [reply]
Yeah.. got it...

**Comment by creepy-pasta101** [comment]
@Yisus7u7.... Atlast... I could compile gobject-introspection.... Here are the debs....
https://github.com/suhan-paradkar/xwayland4tewmux/releases/tag/gobject
It must be available in termux soon....

**Comment by JesusChapman** [comment]
Thanks, I'll try it

**Comment by JesusChapman** [comment]
![Screenshot_20210603-082535](https://user-images.githubusercontent.com/64093255/120652390-5a32d380-c445-11eb-8d81-0b6828b17b64.png)

Apparently there are errors with the gtk module, and that's what I want most

**Reply by creepy-pasta101** [reply]
@Yisus7u7 Try after installing pygobject.. it is  now easily installable from pip

**Reply by JesusChapman** [reply]
I did it a while ago, and it doesn't work

**Reply by creepy-pasta101** [reply]
I am not able to understand the reason for this behaviour... Will check soon

**Reply by JesusChapman** [reply]
Well

**Comment by JesusChapman** [comment]
@suhan-paradkar

**Comment by JesusChapman** [comment]
@suhan-paradkar
